tests/v1/core/test_scheduler.py

```diff
@@ -429,7 +436,7 @@ def test_schedule_concurrent_partial_requests(enable_prefix_caching: bool):
 
 def test_stop_via_update_from_output():
     """Test stopping behavior through update_from_output"""
-    scheduler = create_scheduler()
+    scheduler = create_scheduler(num_speculative_tokens=1)
```

Comment:
This test case will trigger spec decoding metrics recording, so now we need `num_spec_tokens` to be set in the config